app/controllers/messages_controller.rb

```diff
@@ -50,6 +50,31 @@ def destroy
     end
   end
 
+  def update
+    authorize @message
+
+    if @message.update(permitted_attributes(@message).merge(edited_at: Time.zone.now))
+      if @message.valid?
+        begin
+          message_json = create_pusher_payload(@message, "")
+          Pusher.trigger(@message.chat_channel.pusher_channels, "message-edited", message_json)
+        rescue Pusher::Error => e
+          logger.info "PUSHER ERROR: #{e.message}"
+        end
+      end
```

Comment:
This whole block of code can be moved to a service IMHO. The controller looks bloated